### DISCUSSION TITLE
Bad performance with my data

### DISCUSSION THREAD

**Discussion by epicmau5time** [rsuite/rsuite-table#286] [Q&A]
Was wondering if the type of data impacts the performance negatively. Reason I ask is that I used the fake data, duplicated it a ton of times, virtualized the table and added the same amount of columns to mirror my real data. The performance with your fake data is far faster than my data using the same dimensions (number of rows, number of columns).

My data is primarily text based, real-estate data, so think, addresses, number of rooms, sale price, number of stories, etc. No matter what optimizations I try to make, when I use my data it's slower.

**Comment by simonguo** [comment]
The performance and data volume of the table is related to the complexity of the final rendered DOM, not to the data type.
